packages/ra-ui-materialui/package.json

```diff
@@ -75,6 +75,7 @@
         "jsonexport": "^2.4.1",
         "lodash": "~4.17.5",
         "prop-types": "^15.6.1",
+        "query-string": "~5.1.1",
```

Comment:
please relax the version constraint with a `^` instead of `~` to allow npm to avoid duplicate packages.